docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+        "betrokkeneIdentificatie": {
+            "inpBsn": "111222333"
+            },
+        "authenticatieContext": null
+    }
+
+
+
+3. eHerkenning / Without mandate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**eHerkenning - initiator (no branch)**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "betrokkeneIdentificatie": {
+            "innNnpId": "999999999"
+        },
+        "authenticatieContext": {
+            "source": "eherkenning",
+            "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+            "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+        }
+    }
+
+
+**eHerkenning - initiator (branch)**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+            "source": "eherkenning",
+            "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+            "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+        }
+    }
+
+
+4. eHerkenning / With mandate (bewindvoering)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**eHerkenning - initiator**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+            "source": "eherkenning",
+            "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+            "representee": {
+                "identifierType": "bsn",
+                "identifier": "111222333"
+            },
+            "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
```

Comment:
missing trailing comma, it breaks the syntax highlighting :D

Reply:
Added